### PR TITLE
Adds Fontin Sans link to README, fixes typo, adds my name to authors, and deletes duplicate slide

### PR DESCRIPTION
**Pull request by Oddant1**
I added some content to the slides. I didn't mess with any of the technical pieces just some of the background. It is all accurate information to the best of my knowledge but obviously if I'm wrong on anything let me know.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,5 +10,6 @@ Available at https://github.com/drbunsen/drbunsen-beamer
 
 + [XeTeX](http://en.wikipedia.org/wiki/XeTeX)
 + [The Fontin Font](http://www.exljbris.com/fontin.html)
```
Comment (ChrisKeefe) [inline README.md]: @Oddant1, thanks for catching this and adding Fontin Sans. Would you mind looking through the source to confirm whether the Fontin (not-sans) family is used anywhere and removing it if it isn't?

File: getGit.tex [hunk 2]
```diff
@@ -91,51 +91,82 @@ \section{Introduction}
...
+    	\item{Git is a powerful version control system}
+    		\begin{itemize}
+    			\item{Git allows multiple people to easily update the same source}
+    			\item{Git makes keeping tracking of changes easy and robust}
+    		\end{itemize}
+    	\item{Git allows people around the world to collaborate on open and closed source projects}
+   	\end{itemize}
+    \vspace{1cm} % generate some space between title and content
+
+    % Keeping all of this for now as an example in case we need to do something like this
```
Comment (ChrisKeefe) [inline getGit.tex:104]: Perfect! Thanks for thinking ahead.

File: getGit.tex [hunk 3]
```diff
@@ -91,51 +91,82 @@ \section{Introduction}
...
+    \begin{itemize}
+    	\item{Git is a version control system}
+    	\item{GitHub is a site cataloging some git repositories}
+    		\begin{itemize}
+    			\item{It is free to put open source repos on GitHub}
+    			\item{You have to pay to catalog closed source repos}
+    			\item{Not all git repos are catologed on GitHub}
+    			\item{Git repos absolutely do not need to be on GitHub}
+    		\end{itemize}
+    	\item{GitHub gives an easy way to access and manage git repos}
```
Comment (ChrisKeefe) [inline getGit.tex]: "Github gives an easy..." feels a little redundant to me. Cut? 

Also, "tons of open source..." should be "indented" with the github list, not at the same level as Git vs. GitHub points.

File: getGit.tex [hunk 4]
```diff
@@ -91,51 +91,82 @@ \section{Introduction}
...
-
+    \begin{itemize}
+    	\item{Everyone uses git}
+    		\begin{itemize}
+    			\item{Major corporations}
+    			\item{Independent software developers}
+    			\item{Students collaborating on projects}
+    			\item{Many MANY open source projects}
+    		\end{itemize}
+    	\item{Git can easily be used for open or closed source projects making it a diverse and powerful tool}
```
Comment (ChrisKeefe) [inline getGit.tex]: @Oddant1 Please add governments to list of  user groups. https://government.github.com/community/ (No need to add link)

Next time we meet, let's discuss aesthetic for this slide. I have some ideas...

File: getGit.tex [hunk 5]
```diff
@@ -154,23 +185,12 @@ \section{What, Why, and How?}
...
-    \begin{itemize}
-        \item{Windows doesn't git well.}
-        \item{Linux install through package manager}
+        \item{Windows: Install gitbash.}
+        \item{Linux: Install through package manager.}
         \item{Mac: ???}
     \end{itemize}
     \vspace{1cm} % generate some space between title and content
 \end{frame}
-\section{Getting Started with git}
```
Comment (ChrisKeefe) [inline getGit.tex:173]: Why did you drop this?

File: getGit.tex [hunk 6]
```diff
@@ -154,23 +185,12 @@ \section{What, Why, and How?}
...
-\end{frame}
-
-\begin{frame}
-    \frametitle{Git and Operating Systems}
-    \begin{itemize}
-        \item{Windows doesn't git well.}
-        \item{Linux install through package manager}
+        \item{Windows: Install gitbash.}
+        \item{Linux: Install through package manager.}
         \item{Mac: ???}
```
Comment (ChrisKeefe) [inline getGit.tex]: @turanoo What's the install process like for git on a mac?

**Review by ChrisKeefe — COMMENTED** [review]
Nice work, @Oddant1! Thanks so much for putting time into this. I've added some comments and requested changes inline.

File: README.md (hunk 1 above)
Comment (Oddant1) [reply README.md]: I couldn't find any reference to non fontin sans fonts. I went ahead and moved the non sans fontin fonts to my desktop just to be sure and the project still built fine, so I removed it from dependencies.

File: getGit.tex (hunk 3 above)
Comment (Oddant1) [reply getGit.tex]: Done

File: getGit.tex (hunk 4 above)
Comment (Oddant1) [reply getGit.tex]: Done, sounds good

File: getGit.tex (hunk 5 above)
Comment (Oddant1) [reply getGit.tex:173]: I think I was just messing with deleting stuff to see what effect it had on the formatting, and I forgot to add it back. I fixed it now my bad.

File: getGit.tex (hunk 6 above)
Comment (Oddant1) [reply getGit.tex]: Not sure, I'll ask one of my friends with a Mac if he's willing to install git on it. I can't imagine it's very hard, but I don't have access to a Mac to try

File: README.md (hunk 1 above)
Comment (ChrisKeefe) [reply README.md]: :+1:

File: getGit.tex [hunk 7]
```diff
@@ -160,6 +159,7 @@ \section{What, Why, and How?}
     	\item{Everyone uses git}
     		\begin{itemize}
     			\item{Major corporations}
+    			\item{The government}
```
Comment (ChrisKeefe) [inline getGit.tex]: Sorry to nitpick, @Oddant1. "Governments" is more accurate here. US and state governments, but also governmental entities all over the world use git.

**Review by ChrisKeefe — CHANGES_REQUESTED** [review]
Two final tweaks, and this is good to go. Thanks for your patience!

File: getGit.tex (hunk 7 above)
Comment (Oddant1) [reply getGit.tex]: No definitely nitpick. One of the biggest advantages of git here is it makes every single change extremely transparent. My buddy and I looked at installing git on his Mac and as expected it was just a very simple program to download and run. I included the name of the program in this latest commit

**Pull request closed without merging**